source/package.json-e

```diff
@@ -14,26 +14,31 @@
-    "compile-ts-client-stack": "tsc deployment/custom-deployment/lib/cdk-textract-client-stack.ts --target es2018 --module commonjs --allowjs true",
-    "deploy:stack": "yarn compile-ts-stack && AWS_REGION=$npm_package_stack_region cdk deploy -v true",
-    "deploy:client": "yarn compile-ts-client-stack && yarn export && yarn update-client-stack-variables && AWS_REGION=$npm_package_stack_region cdk deploy -v true -a \"node bin/deploy-client-stack.js\" && yarn update-pdf-lambda-code",
+    "bootstrap": "AWS_REGION=$npm_package_stack_region USER_EMAIL=$npm_package_email cdk bootstrap --toolkit-stack-name DocumentUnderstandingCDKToolkit",
+    "compile-ts-stacks": "yarn compile-ts-backend-stack && yarn compile-ts-client-stack",
+    "compile-ts-backend-stack": "tsc lib/cdk-textract-stack.ts --target es2018 --module commonjs --allowjs true",
+    "compile-ts-client-stack": "tsc lib/cdk-textract-client-stack.ts --target es2018 --module commonjs --allowjs true",
+    "deploy:backend": "ISCICD=$npm_package_cicd STACKNAME=$npm_package_stack_stackname AWS_REGION=$npm_package_stack_region USER_EMAIL=$npm_package_email cdk deploy --toolkit-stack-name DocumentUnderstandingCDKToolkit --require-approval never -v true",
+    "deploy:client": "yarn export && yarn update-client-stack-variables && AWS_REGION=$npm_package_stack_region STACKNAME=$npm_package_stack_stackname cdk deploy --toolkit-stack-name DocumentUnderstandingCDKToolkit --require-approval never -v true -a \"node bin/deploy-client-stack.js\"",
     "deploy:setup-user": "AWS_REGION=$npm_package_stack_region node bin/user-setup.js",
-    "deploy:setup-samples": "AWS_REGION=$npm_package_stack_region ./bin/upload-samples.sh",
+    "deploy:setup-samples": "AWS_REGION=$npm_package_stack_region bash ./bin/upload-samples.sh",
     "deploy:show": "AWS_REGION=$npm_package_stack_region node ./bin/client-stack-url.js",
-    "deploy": "yarn deploy:stack && yarn deploy:client && yarn deploy:setup-samples && yarn deploy:show",
+    "deploy-all": "yarn compile-ts-stacks && yarn bootstrap && yarn deploy:backend && yarn deploy:client && yarn deploy:setup-samples && yarn deploy:show",
+    "deploy": "yarn deploy-all && yarn update-pdf-lambda-code",
     "deploy:frontend": "yarn deploy:client && yarn deploy:setup-samples && yarn deploy:show",
-    "destroy": "AWS_REGION=$npm_package_stack_region cdk destroy -v true -a \"node bin/deploy-client-stack.js\" && AWS_REGION=$npm_package_stack_region cdk destroy -v true",
-    "update-client-stack-variables": "bash ./deployment/custom-deployment/bin/stack-variables-to-client.sh",
-    "update-pdf-lambda-code": "bash ./deployment/custom-deployment/bin/update-pdf-lambda-code.sh",
+    "destroy": "STACKNAME=$npm_package_stack_stackname AWS_REGION=$npm_package_stack_region USER_EMAIL=$npm_package_email cdk destroy -v true $npm_package_stack_stacknameStack $npm_package_stack_stacknameClientStack",
```

Comment:
Any reason why the command has been changed to this?
Would it beneficial to get the client stack name from  "node bin/deploy-client-stack.js" as done in yarn deploy:client?